Oqtane.Client/Modules/Controls/ActionDialog.razor

```diff
@@ -83,6 +104,9 @@
     [Parameter]
     public string IconName { get; set; } // optional - specifies an icon for the link - default is no icon
 
+    [Parameter]
+    public EventCallback<string> OnClose { get; set; } // optional - executes a method in the calling component when the Modal is Closed
```

Comment:
I like to introduce event callback in such components as extensibility point, @sbwalker we might need to revise the available components